assets/javascripts/discourse/templates/connectors/discovery-list-container-top/proper_ad.hbs

```diff
@@ -1,5 +0,0 @@
-<div class="proper-ad-unit">
-  <div id="proper-ad-sitepoint_community_home">
-    <script> propertag && propertag.cmd.push(function () { proper_display('sitepoint_community_home'); }); </script>
-  </div>
-</div>
```

Comment:
Please add a new line.

Reply:
The entire file was deleted nothing to add.

Reply:
Ah, yeah. Sorry!